server/src/main/java/org/opensearch/common/util/concurrent/ThreadContext.java

```diff
@@ -213,6 +215,10 @@ public Writeable captureAsWriteable() {
      * if it can't find the task in memory.
      */
     public StoredContext stashWithOrigin(String origin) {
```

Comment:
@cwperks my apologies, I missed that, could you please add javadocs here the same way we did for `markAsSystemContext`? Thank you.

Reply:
yes absolutely. That's my bad. I should have included that in the PR. Good catch.